llvm/lib/ProfileData/InstrProfReader.cpp

```diff
@@ -1261,6 +1261,14 @@ Error IndexedInstrProfReader::readHeader() {
         /*Buckets=*/Start + FrameTableOffset,
         /*Payload=*/Start + FramePayloadOffset,
         /*Base=*/Start, memprof::FrameLookupTrait()));
+
+#if EXPENSIVE_CHECKS
```

Comment:
nit: I think the convention is to use `#ifdef` which is slightly different in meaning to `#if` only.